simplecpp.cpp

```diff
@@ -1839,8 +1839,37 @@ namespace simplecpp {
 }
 
 #ifdef SIMPLECPP_WINDOWS
+
+class RealFileNameMap
+{
+public:
+    bool getRealPathFromCache(std::string const& path, std::string& returnPath)
```

Comment:
Coding style; I prefer that output parameters are pointers. That convention is used in many places in Cppcheck/simplecpp but maybe not all.